bitnami/dataplatform-bp2/README.md

```diff
@@ -96,125 +96,126 @@ The command removes all the Kubernetes components associated with the chart and
-| `kafka.zookeeper.resources.limits`              | Resource limits for zookeeper                      | `{}`                  |
-| `kafka.zookeeper.resources.requests.cpu`        | CPU capacity request for zookeeper                 | `250m`                |
-| `kafka.zookeeper.resources.requests.memory`     | Memory capacity request for zookeeper              | `5120Mi`              |
-| `kafka.zookeeper.affinity.podAntiAffinity`      | Zookeeper anti affinity rules                      | `{}`                  |
-| `kafka.externalZookeeper.servers`               | Array of external Zookeeper servers                | `[]`                  |
+| Name                                                                                      | Description                                                                                                                        | Value                 |
+| ----------------------------------------------------------------------------------------- | ---------------------------------------------------------------------------------------------------------------------------------- | --------------------- |
+| `kafka.enabled`                                                                           | Enable Kafka subchart                                                                                                              | `true`                |
+| `kafka.replicaCount`                                                                      | Number of Kafka brokers                                                                                                            | `3`                   |
+| `kafka.heapOpts`                                                                          | Kafka Java Heap size                                                                                                               | `-Xmx4096m -Xms4096m` |
+| `kafka.resources.limits`                                                                  | Resource limits for Kafka                                                                                                          | `{}`                  |
+| `kafka.resources.requests.cpu`                                                            | CPU capacity request for Kafka nodes                                                                                               | `250m`                |
+| `kafka.resources.requests.memory`                                                         | Memory capacity request for Kafka nodes                                                                                            | `5120Mi`              |
+| `kafka.affinity.podAntiAffinity.requiredDuringSchedulingIgnoredDuringExecution`           | !!!! Kafka anti affinity rules                                                                                                     | `{}`                  |
```

Comment:
`!!!! ...` ???

Reply:
Removed, I was flagging inconsistencies with it and slipped 🤦